app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:gravity="center"
+    android:orientation="vertical"
+    >
+
+    <ImageView
+        android:id="@+id/vLoginCover"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:src="@drawable/login_cover"
+        android:scaleType="centerCrop"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        tools:ignore="MissingConstraints"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="@id/vLoginCover"
+        app:layout_constraintStart_toStartOf="@id/vLoginCover"
+        app:layout_constraintEnd_toEndOf="@id/vLoginCover"
+        android:background="@drawable/gradient"
+
+        />
+
+    <ImageView
+        android:id="@+id/vLoginLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/wolox_logo_large"
+        app:layout_constraintBottom_toTopOf="@id/vEmailInputLogin"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="packed" />
+
+    <EditText
+        android:id="@+id/vEmailInputLogin"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="20dp"
```

Comment:
Remember to add all the "hardcoded values" to where it should be. In this case, to dimens. In fact, those dimens are already in the dimens file, so you can just use what is already defined.
Another thing here is that it is better to use marginStart and marginEnd when you can. I recommend you to look up in the internet why marginStart and marginEnd. There some apps, like the one we are working now with Lucas, that is build from API16 and beyond, and we can't use marginStart and marginEnd because they are to be implemented since API17.